<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Serial</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_19">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPlainTextEdit" name="txt_receber"/>
          </item>
          <item>
           <widget class="QComboBox" name="cb_serial"/>
          </item>
          <item>
           <widget class="QComboBox" name="cb_baudrate"/>
          </item>
          <item>
           <widget class="QPushButton" name="bt_open">
            <property name="text">
             <string>Open</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_close">
            <property name="text">
             <string>Close</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_refresh">
            <property name="text">
             <string>Refresh</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_clear">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>UDPReciever</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPlainTextEdit" name="plainTextEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="bt_send">
          <property name="text">
           <string>Send Hello UDP</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>JoyStick</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QGraphicsView" name="graphicsViewJoy">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>360</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/joy.png) 0 0 0 0 stretch stretch;
border-width: 1px black solid;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lb_values">
          <property name="text">
           <string>0;0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Encoders</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="1">
         <widget class="QLineEdit" name="ln_encoder">
          <property name="text">
           <string>25000</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="bt_encoder">
          <property name="text">
           <string>Enviar</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Frente Até:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Trás Até:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="ln_encoder_2">
          <property name="text">
           <string>25000</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="bt_encoder2">
          <property name="text">
           <string>Enviar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
